.bazelrc

```diff
@@ -27,7 +27,7 @@ build --incompatible_strict_action_env
 build --host_force_python=PY3
 build --java_runtime_version=remotejdk_11
 build --tool_java_runtime_version=remotejdk_11
-#build --platform_mappings=""                                                               # unique
+build --platform_mappings=""                                                               # unique
```

Comment:
This line was previously marked as "# unique" and commented out on purpose, since we don't need platform mappings in Nighthawk.

Do we need to uncomment it?

Reply:
It appears that it works this line commented as well. Address in #851